### PR TITLE
add signaling to connections and expose sendSignal API via OTSession ref

### PR DESCRIPTION
**Pull request by msach22**
### Solves issue(s)

https://github.com/opentok/opentok-react-native/issues/195
https://github.com/opentok/opentok-react-native/issues/132

### CODE REVIEW

File: src/OTSession.js [hunk 1]
```diff
@@ -35,8 +35,7 @@ export default class OTSession extends Component {
     const updateSessionProperty = (key, defaultValue) => {
       if (shouldUpdate(key, defaultValue)) {
         const value = useDefault(this.props[key], defaultValue);
-        const signalData = sanitizeSignalData(value);
-        OT.sendSignal(signalData.signal, signalData.errorHandler);
+        this.sendSignal(value)
```
Comment (ggoldens) [inline src/OTSession.js]: Minor request: add the trailing semicolon.

File: src/OTSession.js [hunk 2]
```diff
@@ -58,8 +57,7 @@ export default class OTSession extends Component {
               sessionInfo,
             });
             logOT(credentials.apiKey, credentials.sessionId, 'rn_on_connect', session.connection.connectionId);
-            const signalData = sanitizeSignalData(this.props.signal);
-            OT.sendSignal(signalData.signal, signalData.errorHandler);
+            this.sendSignal(this.props.signal)
```
Comment (ggoldens) [inline src/OTSession.js]: Same here.

File: src/OTSession.js (hunk 1 above)
Comment (msach22) [reply src/OTSession.js]: updated!

File: src/OTSession.js (hunk 2 above)
Comment (msach22) [reply src/OTSession.js]: updated!

**Comment by ggoldens** [comment]
👍 LGTM!

**Pull request merged**